docs/api-reference/stats-widget/stats-widget.md

```diff
@@ -25,20 +24,25 @@ statsWidget.update();
 
 ### constructor
 
-`new StatsWidget(statsInstance, styles)`
-
-* `statsInstance` (`Stats`) - a probe.gl [Stats](/docs/api-reference/log/stats.md) instance.
-* `styles` (Object, optional)
-  - `container` (DOMElement) - an element to use as the container of the widget. If not provided, a new `div` will be appended to the document as the container.
-  - `containerStyle` (String) - css text for the container. Only applied if using the default container.
-  - `width` (Number) - width of the widget, in pixels
-  - `color` (String) - css color for the text
-  - `background` (String) - css color for the background
-  - `padding` (Array) - [horizontal, vertical] padding, in pixels
-  - `fontFamily` (String) - css font family for the text
-  - `headerSize` (Number) - font size of the header
-  - `fontSize` (Number) - font size of the body
-  - `lineSpacing` (Number) - spacing between lines, in pixels
+`new StatsWidget(stats, opts)`
+
+* `stats` (`Stats`) - a probe.gl [Stats](/docs/api-reference/log/stats.md) instance.
+* `opts` (`Object`, optional)
+  - `title` (`String`) - header text for the widget. Defaults to the `id` of the `Stats` object.
+  - `framesPerUpdate` (`Number`) - number of times `update` must be called before the widget is re-rendered. Allows the application
+    to call `update` each frame with re-renders occuring at a slower rate.
+  - `container` (DOMElement) - DOM element to use as container for the widget. Will be created internally if not provided.
+  - `css` (`Object`) - css properties to apply to the container `div` of the widget.
```

Comment:
Call it `styles` and nest sub class styles as `styles.header` etc.?

Reply:
I was thinking about this, but found that because the other elements inherit CSS from the container, that was the only one I was using. And because you almost always want to set the container style for positioning, I wanted to make that usage easy. Just wanted to keep the other two for cases where the app wants to override the defaults (mainly for header font size and item indentation).